views/index.ejs

```diff
@@ -1,9 +1,9 @@
 <!doctype html>
 <html>
 <head>
+    <link rel="stylesheet" type="text/css" href="/css/index.css"/> <!-- This stylesheet is included twice because it needs to be loaded before AND after for the background animations to work -->
```

Comment:
If this doesn't happen, then the styles.css from the template is applied first, and the animation won't work for the html since it'll start the animation from a different background color.

Reply:
If a double import isn't desirable, another fix could be just to the end state. The animation would be broken but there wouldn't be a mismatch in the final states of the html and body background colors.

Reply:
You could add `important` to the css to overwrite styles, could that help avoid double import? Double import is typically less favored practice

Reply:
Let me know your thoughts, I can help you check in afterwards.